e2e/e2e-test.js

```diff
@@ -51,18 +59,67 @@ async function run() {
         }
     })
 
+    const initialTbtcAccountBalance = await getTbtcAccountBalance(
+        web3,
+        tbtc,
+        web3.eth.defaultAccount
+    )
+
+    console.log(
+        `Initial TBTC balance for account ${web3.eth.defaultAccount} ` +
+        `is: ${initialTbtcAccountBalance}`
+    )
 
     const deposits = []
     for (let i = 1; i <= depositsCount; i++) {
         console.log(`\nStarting deposit number [${i}]...\n`)
         const deposit = await createDeposit(tbtc, satoshiLotSize)
         deposits.push(deposit)
+
+        assertMintedTbtcAmount(web3, deposit, tbtcDepositAmountMinusSignerFee)
+
         console.log(`\nDeposit ${deposit.address} has been created successfully.`)
     }
 
+    const afterDepositsTbtcAccountBalance = initialTbtcAccountBalance.add(
+        web3.utils.toBN(depositsCount).mul(
+            web3.utils.toBN(tbtcDepositAmountMinusSignerFee)
+        )
+    )
+
+    console.log(
+        `TBTC balance for account ${web3.eth.defaultAccount} after ` +
+        `performing deposits should be: ${afterDepositsTbtcAccountBalance}. ` +
+        `Checking assertion...`
+    )
+
+    await assertTbtcAccountBalance(
+        web3,
+        tbtc,
+        web3.eth.defaultAccount,
+        afterDepositsTbtcAccountBalance
+    )
+
     console.log(`\nStarting redemption of the first deposit...\n`)
     const message = await redeemDeposit(tbtc, deposits[0].address, btcAddress)
     console.log(`\nRedemption outcome: ${message}\n`)
+
+    const afterRedemptionTbtcAccountBalance = afterDepositsTbtcAccountBalance.sub(
+        web3.utils.toBN(tbtcDepositAmount)
+    )
+
+    console.log(
+        `TBTC balance for account ${web3.eth.defaultAccount} after ` +
+        `performing redemption should be: ${afterRedemptionTbtcAccountBalance}. ` +
+        `Checking assertion...`
+    )
+
+    await assertTbtcAccountBalance(
+        web3,
+        tbtc,
+        web3.eth.defaultAccount,
+        afterRedemptionTbtcAccountBalance
+    )
```

Comment:
I'd add bitcoin address balance check after redemption.

Reply:
Do you think it's achievable? Can you cover it in this PR or do we want to handle it in a separate PR?

Reply:
May require some juggling with BTC wallets but I'll try.